pinot-controller/src/main/java/org/apache/pinot/controller/ControllerStarter.java

```diff
@@ -219,7 +221,7 @@ public void start() {
   private void setUpHelixController() {
     // Register and connect instance as Helix controller.
     LOGGER.info("Starting Helix controller");
-    _helixControllerManager = HelixSetupUtils.setup(_helixClusterName, _helixZkURL, _instanceId);
+    _helixControllerManager = HelixSetupUtils.setupHelixController(_helixClusterName, _helixZkURL, _instanceId);
```

Comment:
It's not just setting up the controller, it also creates the cluster if needed as well.

Reply:
True, while setting up Pinot controller also create the cluster, so keep them the same

Reply:
Nope, setting up Pinot controller won't create the cluster; it'll throw exception if cluster is not found. The cluster has to be created by Helix controller.